From cad56af2295104c19c6476655fde53fec335ebb0 Mon Sep 17 00:00:00 2001
From: Xiang Wang <xiang.a.wang@intel.com>
Date: Thu, 15 Jan 2015 23:22:42 +0800
Subject: [PATCH] i2c-designware: reduce logs in dw_i2c_acpi_space_handler

Change to use pr_debug to print the debug messages
in dw_i2c_acpi_space_handler.

Change-Id: I3a577c93b2a3d34a9c9a6227f4825d4f8e59d7eb
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-15439
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-930
Signed-off-by: Xiang Wang <xiang.a.wang@intel.com>
(cherry picked from commit 2e92e4e3a5be3b8e8f18471d44f1c388001daaf2)

Signed-off-by: Brett T. Warden <brett.t.warden@intel.com>
---
 drivers/i2c/busses/i2c-designware-platdrv.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/i2c/busses/i2c-designware-platdrv.c b/drivers/i2c/busses/i2c-designware-platdrv.c
index bc8154d..91093a4 100644
--- a/drivers/i2c/busses/i2c-designware-platdrv.c
+++ b/drivers/i2c/busses/i2c-designware-platdrv.c
@@ -106,12 +106,12 @@ dw_i2c_acpi_space_handler(u32 function, acpi_physical_address address,
 	if (sb->type != ACPI_RESOURCE_SERIAL_TYPE_I2C)
 		return AE_BAD_PARAMETER;
 
-	pr_info("%s: Found I2C Resource type, addr %d\n",
+	pr_debug("%s: Found I2C Resource type, addr %d\n",
 				__func__, sb->slave_address);
 	target = sb->slave_address;
 
 	length = acpi_get_serial_access_length(accessor_type, info->access_length);
-	pr_info("%s: access opeation region, addr 0x%x operation %d len %d\n",
+	pr_debug("%s: access opeation region, addr 0x%x operation %d len %d\n",
 		__func__, addr, function, length);
 
 	if (!value64)
-- 
1.7.9.5

